<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>1161</width>
     <height>701</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="requestEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите SQL-запрос...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="executeRequestButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Выполнить</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>841</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="statusTextBrowser">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Статус</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>80</y>
      <width>291</width>
      <height>581</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Структура</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>1171</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="exportAsAction">
     <property name="title">
      <string>Экспортировать...</string>
     </property>
     <addaction name="action_SQL"/>
     <addaction name="action_CVS"/>
     <addaction name="action_TVS"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="exportAsAction"/>
    <addaction name="saveAsAction"/>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="editAction"/>
    <addaction name="deleteAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="editMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <action name="openAction">
   <property name="text">
    <string>Открыть...</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>Сохранить...</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="action_SQL">
   <property name="text">
    <string>В базу данных SQL...</string>
   </property>
  </action>
  <action name="action_CVS">
   <property name="text">
    <string>В базу данных CVS...</string>
   </property>
  </action>
  <action name="action_TVS">
   <property name="text">
    <string>В базу данных TVS...</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Размер таблицы...</string>
   </property>
  </action>
  <action name="editAction">
   <property name="text">
    <string>Изменение данных...</string>
   </property>
  </action>
  <action name="deleteAction">
   <property name="text">
    <string>Удаление данных...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
